core/node/rpc/server.go

```diff
@@ -571,6 +571,9 @@ func (s *Service) initHandlers() {
 	ii := []connect.Interceptor{}
 	if s.otelConnectIterceptor != nil {
 		ii = append(ii, s.otelConnectIterceptor)
+		if i, err := otelconnect.NewInterceptor(); err == nil {
```

Comment:
It looks like this is redundant: otel connect interceptor is created and configured in otel.go and appended on the previous line?